src/drivers/mct/shr/seq_timemgr_mod.F90

```diff
@@ -907,6 +912,11 @@ subroutine seq_timemgr_clockInit(SyncClock, nmlfile, restart, restart_file, pioi
             RefTime = CurrTime,            &
             alarmname = trim(seq_timemgr_alarm_restart))
 
+       call seq_timemgr_alarmGet(SyncClock%EAlarm(n,seq_timemgr_nalarm_restart), IntSec=RestInterval)
+       if(n==1 .or. RestInterval < minRestInterval) then
```

Comment:
Since Fortran doesn't use short-circuit boolean operators, I believe some compilers will complain about uninitialized variables when executing this in debug mode. It should be reworked to look like what you did for setting minStopTime.